docs/testreport/scripts/bsi_test_report.py

```diff
@@ -88,9 +88,11 @@ def assertions(self):
         return sum(asserts)
 
     def _format_command(self, command):
+        # U+200B is a zero-width space: i.e. it is not rendered but allows line
+        #        breaks to appear at this location.
         cmd = re.sub(r"^.+botan-test", "botan-test", command) \
             .replace('\\', '/') \
-            .replace(';', ',:raw-latex:`\\allowbreak` ') \
+            .replace(';', ',\u200B') \
```

Comment:
Note that this `raw-latex` role is defined in `docs/testreport/testreport_preamble.rst`. If we no longer use it, we should probably also delete the role.

Reply:
# 🤦 

Oh, indeed! That also explains why it stopped working. The whole `testreport_preamble.rst` isn't used anymore with the new document structure. I'll take care of it tmrw. Thanks.

Reply:
Done. 7eb2c2c.